src/templates/web/base.html

```diff
@@ -0,0 +1,241 @@
+
+    <title>{% block titulo %} {% endblock %}</title>
+
+</head>
+
+<body>
+    {% if messages %}
+    <ul class="messages">
+        {% for message in messages %}
+        <li{% if message.tags %} class="{{ message.tags }}" {% endif %}>
+            {% if message.level == DEFAULT_MESSAGE_LEVELS.ERROR %}Important: {% endif %}
+            {{ message }}
+            </li>
+            {% endfor %}
+    </ul>
+    {% endif %}
+
+    {% block navbar %}
+    <div class="container">
+        <div class="row align-items-center justify-content-center">
+            <div class="col-lg-2 col-md-12">
+                <div class="logo">
+                    <a class="navbar-brand" href="#"><img src="{% static 'img/manga.png' %}" width="150" height="100"></a>
+                </div>
+            </div>
+
+            <form class="col-lg-10 col-md-12">
+                <div class="row">
+                    <div class="col-lg-8">
+                        <input class="form-control" type="search" placeholder="Buscar productos" aria-label="Search">
+                    </div>
+                    <div class="col-lg-4">
+                        <button type="button" class="btn btn-primary btn-block">
+                            <i class="fa-solid fa-cart-plus"></i>
+                        </button>
+                        <button type="button" class="btn btn-primary btn-block">
+                            <i class="fa-solid fa-user"></i>
+                        </button>
+                    </div>
+                </div>
+            </form> 
+        </div>
+    </div>
+
+    <br>
+    <nav class="navbar navbar-expand-lg navbar navbar-dark bg-dark">
+        <div class="container-fluid">
+
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+                    <li class="nav-item">
+                        <a class="nav-link active" aria-current="page" href="#">Inicio</a>
+                    </li>
+                    <li class="nav-item dropdown">
+                        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button"
+                            data-bs-toggle="dropdown" aria-expanded="false">
+                            Categorías
+                        </a>
+                        <ul class="dropdown-menu " aria-labelledby="navbarDropdown ">
+                            
+                            {% for i in categoria%}
+                            <li><a class="dropdown-item" href="#">{{i}}</a></li>
+                            {%endfor%}
```

Comment:
```suggestion
                            {% endfor %}
```

separar la lógica de los corchetes.